packages/cli/src/commands/ui/create/vue.ts

```diff
@@ -16,7 +17,7 @@ import {getPackageVersion} from '../../../lib/utils/misc';
 
 export default class Vue extends Command {
   static templateName = '@coveo/vue-cli-plugin-typescript';
-
+  static requiredNodeVersion = '12.21.0';
```

Comment:
I would add (here + for the react command I guess), some link to their documentation that explains this choice.

So that the next guy that needs to modify this knows "where to look for".

Reply:
- [ ] Add doc link to Vue requirements